src/libs/actions/Policy.js

```diff
@@ -265,7 +265,7 @@ function update(policyID, values) {
 
             const updatedValues = {...values, ...{isPolicyUpdating: false}};
             Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, updatedValues);
-            Navigation.dismissModal();
+            Growl.show(translateLocal('workspace.editor.growlMessageOnSave'), CONST.GROWL.SUCCESS, 5000);
```

Comment:
Can we go with 3000 ms here? I think this can be a relatively short message

```suggestion
            Growl.show(translateLocal('workspace.editor.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
```